app/views/observation_accuracy_experiments/_methods.html.haml

```diff
@@ -0,0 +1,103 @@
+      %br
+      ="Mean number of candidate validators per sample observation: #{@mean_validator_count}"
+      %div{style: "clear: both;"}
+.row
+  .col-xs-12
+    %h3 Validator candidate criteria
+    .container2
+      %img{src: "https://static.inaturalist.org/wiki_page_attachments/3708-original.png", style: "width:500px;float: right;clear: right;margin: 0px 40px 0px 60px"}
+      %p
+        If an identifier had made at least 3 improving identifications on a taxon, we considered them qualified to validate that taxon. Improving identifications are the first suggestion of a taxon that the community subsequently agrees with.
+      %p
+        For example, Identifier 1 adds an ID of A to an observation. If Identifier 2 later adds a leading ID of B, Identifier 1's ID on A becomes an improving ID. If Identifier 3 later adds a supporting ID to B, Identifier 2's ID on B becomes an improving ID.
+      %div{style: "clear: both;"}
+.row
+  .col-xs-12
+    %h2 Step2: Validating samples
+.row
+  .col-xs-6
+    %p= "We contacted candidate validators and asked them to add identifications to their subsamples. The graph to the right groups sample observation by the number of validators."
+    = "Mean number of validators per sample observation: #{@mean_validators_per_sample}"
+    %br
+    = "Number of participating validators: #{@experiment.responding_validators} (#{ ( @experiment.responding_validators /  @candidate_validators.to_f * 100 ).round}%)"
+    %br
+    = "Percent of sample validated: #{( @experiment.validated_observations / @experiment.sample_size.to_f * 100 ).round}%"
+
+  .col-xs-6
+    .barplot-container
+      - scale_factor = @validators_per_sample_ylim / 100 
+      %svg{ width: "#{50 + 54 * @validators_per_sample.count}", height: "150" }
+        %g{ transform: "translate(40,10)" }
+          %g.y.axis
+            - (0..5).each do |tick_value|
+              - tick_position = tick_value * 20
+              %g.tick{ transform: "translate(0,#{tick_position})", style: "opacity: 1;" }
+                %line{ x2: "-6", y2: "0" }
+                %text{ dy: ".32em", x: "-9", y: "0", style: "text-anchor: end;" } #{( 100 - tick_value * 20 ) * scale_factor}
+            %path.domain{ d: "M-6,0H0V100H-6" }
+            %text{ transform: "rotate(-90)", y: "6", dy: ".71em", style: "text-anchor: end;"} Frequency
+
+          %g.x.axis{ transform: "translate(0,100)" }
+            - @validators_per_sample.keys.each_with_index do |label, index|
+              %g.tick{ transform: "translate(#{35 + 54 * index},0)", style: "opacity: 1;" }
+                %line{ y2: "6", x2: "0" }
+                %text{ dy: ".71em", y: "9", x: "0", style: "text-anchor: middle;" }= label
+
+          - @validators_per_sample.values.each_with_index do |values, index|
+            %a{ href: @explorable ? FakeView.observations_url( verifiable: "any", place_id: "any", id: values.join( "," ) ) : '#' }
```

Comment:
no need to reference FakeView in a view as they have access to URL helpers directly